src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -0,0 +1,348 @@
+for live streaming and video on demand for the web.
+
+    ffmpeg -i myvideo.mp4 -c:a copy -b:v 8M -c:v copy -f hls -hls_time 10 \
+           -hls_list_size 0 myvideo.m3u8
+
+## Clear Key Encryption
+
+### Create a key
+
+The same method for creating a key may be used with both DASH and HLS. The
+following will create a key made of 16 hex values.
+ 
+    openssl rand 16 > media.key
+    
+This command creates a file with white space and new line characters, which are
+not allowed by Shaka Packager. You can pipe the output to some other commands
+before dumping it to a file.
+
+    openssl rand 16 | tr '\n' ' ' | sed 's/ //g' > media.key
+
+### Encrypt
+
+For the `-key` flag can be the key created earlier and stored in the media.key
+file. However, when entering it on the command line, be sure to remove the
+whitespace from the key. For the `-key_id` flag repeat the key value.
+
+> **Note:** Technically, the key ID is supposed to be either the first 8 OR the
+> first 16 hex digits of the key. Since packager requires the key to be 16
+> digits and does not allow a 32 digit key, both flags use the same value.
+
+ 
+    packager \
+      input=myvideo.mp4,stream=audio,output=glocka.m4a \
+      input=myvideo.mp4,stream=video,output=glockv.mp4 \
+      --enable_fixed_key_encryption --enable_fixed_key_decryption \
+      -key INSERT_KEY_HERE -key_id INSERT_KEY_HERE \
+
+### Create a key information file
+
+To encrypt for HLS you need a key information file in addition to a key file. A
+key information file has the following format.
+ 
+    key URI
+    key file path
+ 
+For example:
+ 
+    https://example.com/media.key
+    media.key
+
+### Encrypt for HLS
+
+    packager \
+      'input=input.mp4,stream=video,segment_template=output$Number$.ts,playlist_name=video_playlist.m3u8' \
+      'input=input.mp4,stream=audio,segment_template=output_audio$Number$.ts,playlist_name=audio_playlist.m3u8,hls_group_id=audio,hls_name=ENGLISH' \
+      --hls_master_playlist_output="master_playlist.m3u8" \
+      --hls_base_url="http://localhost:1000/"
+ 
+This command will accept a key with either 16 or 32 characters.
+ 
+    ffmpeg -i myvideo.mov -c:v libx264 -c:a aac -hls_key_info_file key_info myvideo.m3u8
+
+## Widevine Encryption
+
+Everything in this command except the name of your files and the `--content-id`
+flag should be copied exactly from the example. The `--content-id` is 16 or 32
+random hex digits.
+ 
+    packager \
+      input=glocken.mp4,stream=video,output=enc_glocken.mp4 \
+      input=myvide.mp4,stream=audio,output=enc_myaudio.m4a \
+      --enable_widevine_encryption \
+      --key_server_url "https://license.uat.widevine.com/cenc/getcontentkey/widevine_test" \
+      --content_id "fd385d9f9a14bb09" --signer "widevine_test" \
+      --aes_signing_key "INSERT_KEY_HERE" \
+      --aes_signing_iv "INSERT_KEY_HERE"
+
+## All Together Now
+
+### DASH/webm with Shaka Packager
+
+Not all steps are possible with Shaka Packager.
```

Comment:
When you say this (more than once), you should explicitly call out what you will need FFMPEG for.  "Not all steps are possible with Shaka Packager - we'll first use FFMPEG to convert the file type and codec."